e2e/pods_test.go

```diff
@@ -77,10 +78,6 @@ func TestPodLifecycle(t *testing.T) {
 	// check container logs
 	t.Log("get container logs ....")
 	cmd = kubectl("logs", "pod/vk-e2e-hpa", "-c", "hpa-example", "--namespace=vk-test")
-	out, err = cmd.CombinedOutput()
```

Comment:
Why did we remove lines 80-83

Reply:
this looks like a merge error. The merge conflicts were all over the place. I had moved the init container tests outside this file to avoid the same issue in the future.